### DISCUSSION TITLE
ACME Server SSL Error

### DISCUSSION THREAD

**Discussion by prueckls** [ZoeyVid/NPMplus#1227] [Q&A] [answered]
Unfortunately, there is no longer an option to ignore the SSL certificate for the ACME server, as was previously possible in the cli.ini file in older versions. This change has created significant challenges for my current use case. As a result, I am unable to use the current version effectively and was forced to revert to an earlier version where this option was still available. This limitation could be problematic for users who need flexibility in handling SSL certificates for development or testing environments.

**Comment by Zoey2936** [comment]
can you maybe tell me which options you need to change to make it work and why?

**Comment by prueckls** [comment]
I am using NPMplus with my own PKI and self-signed certificates, so I need an option equivalent to no-verify-ssl = true, as was available in older versions within the config.ini file.

**Comment by Zoey2936** [comment]
you can either open a PR to add an env option to add this flag or you add a prerun scripts (see readme), to modify the certbot config file in /etc/certbot.ini (or just mount a diffrent file to this place)

**Comment by prueckls** [comment]
Thank you for your suggestion. I mounted my own certbot.ini file, and now it's working again. Thanks!

**Comment by Zoey2936** [comment, marked as answer]
The next release will contain and env option to change this
